Lib/multiprocessing/queues.py

```diff
@@ -17,7 +17,6 @@
 import types
 import weakref
 import errno
-import ctypes
 
 from queue import Empty, Full, ShutDown
```

Comment:
Due from splitting PR, `ShutDown` exception is not more included  in `lib/queue.py`. So here we have to define it, here.

Reply:
```py
from queue import Empty, Full
try:
    from queue import ShutDown
except ImportError:
    class ShutDown(Exception):
        pass
```

Reply:
That's why I say

> Depends on #104230

in the PR description. I think the `ShutDown` exception class is the only thing the PR depends on.

Reply:
May be I miss something, but I think that depends on https://github.com/python/cpython/pull/104225, don't you ? 

As CI failed, if you agree, would you update file `lib/multiprocessing/queues.py` with my proposal ?

Reply:
> May be I miss something, but I think that depends on https://github.com/python/cpython/pull/104225, don't you ?

... yup. PR description updated

> As CI failed, if you agree, would you update file `lib/multiprocessing/queues.py` with my proposal ?

Instead, I'll just add `ShutDown` to threading

Reply:
I hope https://github.com/python/cpython/pull/104225 will be accepted and merged before this one ;-)